docs/packages/definitions.md

```diff
@@ -0,0 +1,15 @@
+# Definitions
+
+This document provides definitions of some package terminology used in the Tanzu framework.
+To learn about the Carvel package concepts visit this [page](https://carvel.dev/kapp-controller/docs/latest/packaging)
+
+## Management package
+
+Any package that performs the role of managing and operating a workload load cluster is a Management package.
```

Comment:
Question: we might want to remove the extra `load` from `workload load cluster` in above.

Reply:
thanks for catching that :D